docs/howto/gathering_info/exploitation.md

```diff
@@ -0,0 +1,29 @@
+# Gathering Information About Exploitation
+```python exec="true" idprefix=""
+from ssvc.decision_points.ssvc.exploitation import LATEST
+from ssvc.doc_helpers import example_block
+
+print(example_block(LATEST))
+```
+
+[@householder2020historical] presents a method for searching the GitHub repositories of open-source exploit databases.
```

Comment:
Replace the bib ref with a link to the actual paper with title `[Historical Analysis of Exploit Availability Timelines](https://dl.acm.org/doi/10.5555/3485754.3485760)`